rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -334,6 +298,107 @@ public static LockWhitelist deserializeLockWhitelist(byte[] data, NetworkParamet
         return new LockWhitelist(deserializeBtcAddresses(data, parameters));
     }
 
+    // A ReleaseRequestQueue is serialized as follows:
+    // [address_1, amount_1, ..., address_n, amount_n]
+    // with address_i being the encoded bytes of each btc address
+    // and amount_i the RLP-encoded biginteger corresponding to each amount
+    // To preserve order amongst different implementations of lists,
+    // entries are first sorted on the lexicographical order of the
+    // address bytes and then by the amount ascending
+    // (see ReleaseTransactionSet.Entry.DESTINATION_AMOUNT_COMPARATOR)
+    public static byte[] serializeReleaseRequestQueue(ReleaseRequestQueue queue) {
+        List<ReleaseRequestQueue.Entry> entries = queue.getEntries();
+        entries.sort(ReleaseRequestQueue.Entry.DESTINATION_AMOUNT_COMPARATOR);
```

Comment:
sort of queues is not necessary, and we can preserver the order of it without it

Reply:
Good point. Fixed.